onnx_tf/handlers/backend/depth_to_space.py

```diff
@@ -27,3 +27,38 @@ def version_1(cls, node, **kwargs):
         cls.make_tensor_from_onnx_node(
             node, attrs=attrs, c_first_cuda_only=True, **kwargs)
     ]
+
+  @classmethod
+  def version_11(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    x_rank = len(x.get_shape())
+    storage_format, compute_format = get_data_format(x_rank)
+    attrs = copy.deepcopy(node.attrs)
+    attrs["data_format"] = storage_format
+    mode = attrs.get("mode", "DCR")
+    block_size = attrs.get("blocksize")
+
+    if mode == "CRD":
+      # need native computation
+      bsize = tf.constant(block_size)
```

Comment:
It looks like you don't need `tf.constant` here?
It seems like you can just use python variables throughout.